docs/source/changelog.rst

```diff
@@ -42,6 +42,15 @@ Config system
 #. Implement a :code:`manim.cfg` config file system, that consolidates the global configuration, the command line argument parsing, and some of the constants defined in :code:`constants.py`
 #. Added utilities for manipulating Manim’s :code:`.cfg` files.
 #. Added a subcommand structure for easier use of utilities managing :code:`.cfg` files
+#. Also some variables have been moved from constants.py to the new config system:
```

Comment:
I think you don't want this line to be printed bold; it is currently rendered as a definition list. (Although this missing newline is actually missing several times throughout the documentation... if you are at it, I'd appreciate if you could add it for the other nested lists as well. :-)

```suggestion
#. Also some variables have been moved from ``constants.py`` to the new config system:

```

Reply:
Done, also fixed it in the other nested list